app/containers/TextInput/FormTextInput.tsx

```diff
@@ -91,7 +93,9 @@ export const FormTextInput = ({
 	return (
 		<View style={[styles.inputContainer, containerStyle]}>
 			{label ? (
-				<Text style={[styles.label, { color: colors.titleText }, error?.error && { color: colors.dangerColor }]}>{label}</Text>
+				<Text style={[styles.label, { color: colors.titleText }, labelStyle, error?.error && { color: colors.dangerColor }]}>
```

Comment:
Can you check with design if this different label style is really needed?
We might be creating fragmentation without a good reason.

Reply:
We don't need this fragmentation!